### PR TITLE
Added Wombat Facts

### PR DESCRIPTION
**Pull request by im-sanyam**
Added about another animal - Wombat

### CODE REVIEW

**Comment by im-sanyam** [comment]
Sure I will get to it ASAP.

**Comment by im-sanyam** [comment]
I gues this will do,

File: animalfacts.py [hunk 1]
```diff
@@ -369,6 +369,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('whale', '\swhales?\s', reddit, WHALE_FACTS, comment)
     botengine('wolf', '\swol(f|ves)\s', reddit, WOLF_FACTS, comment)
     botengine('wolverine', '\swolverines?\s', reddit, WOLVERINE_FACTS, comment)
+    botengine('wombat', '\swombatss?\s', reddit, WOMBAT_FACTS, comment)
```
Comment (mpentler) [inline animalfacts.py]: @joel-g Will this work? Does it need the final "s" removing in wombats?

Comment (im-sanyam) [reply animalfacts.py]: I guess so..otherwise can you suggest me the change if possible ?..Thanks

Comment (mpentler) [reply animalfacts.py]: I am reasonably sure it should be '\swombats?s\' so that it matches the other animals. Take out the extra S?

Comment (mpentler) [reply animalfacts.py]: I'll fix this myself.

**Pull request closed without merging**